### DISCUSSION TITLE
Regarding PD elastic_plastic model

### DISCUSSION THREAD

**Discussion by MEERAGNAIR** [PeriHub/PeriLab.jl#6] [Q&A] [answered]
Hi Sir, Myself Meera, a Research Scholar from IIT Guwahati, India. I'm working in the field of structural engineering. I came to see your code in Julia recently. I would like to get the code on the elastic_plastic model. The code for the same here is not complete in its raw form. Kindly help me with the same.

**Comment by CWillberg** [comment, marked as answer]
Dear Meera,

 

the code is complete for the correspondence part. You can call it in the yaml, by combining the elastic Correspondence and the Plastic Correspondence model. You can do that with a +. You can find an example under test/full_scale_tests/test_correspondence_elastic_plastic

 

I hope it helps you. If you have further questions, feel free to contact us.

 

Christian

 

Von: MEERAGNAIR ***@***.***> 
Gesendet: Donnerstag, 22. Februar 2024 13:23
An: PeriHub/PeriLab.jl ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [PeriHub/PeriLab.jl] Regarding PD elastic_plastic model (Discussion #6)

 

Hi Sir, Myself Meera, a Research Scholar from IIT Guwahati, India. I'm working in the field of structural engineering. I came to see your code in Julia recently. I would like to get the code on the elastic_plastic model. The code for the same here is not complete in its raw form. Kindly help me with the same.

**Reply by MEERAGNAIR** [reply]
Thank you so much for your reply. Ill work on it.
